src/components/ArticlesNav.js

```diff
@@ -0,0 +1,48 @@
+import { Link, useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import { fetchTopics } from "../api/topics";
+
+function ArticlesNav({ currentTopic, setCurrentTopic }) {
```

Comment:
If not being utilised in other components it may be best to push down the  state into this component, currentTopic could then be set in the useEffect. 
Changes to the active topic could also be handled by a onClick function which then changes the active class of the current topic selected.
Also currentTopic would then also be the same as topic in your useParams which you've declared on line 16